src/holon/services/query_and_convert.py

```diff
@@ -108,11 +110,11 @@ def costs(config):
             for key, value in cost_components.items():
                 span.set_data("etm_output_cost_" + key, value)
 
-        yield ("costs", sum(cost_components.values()))
+        yield (QueryCovertModuleType.COST, sum(cost_components.values()))
 
         # Calculate depreciation costs for each actor
         yield (
-            "cost_benefit",
+            QueryCovertModuleType.COSTBENEFIT,
```

Comment:
I don't think this is right but I've spent to much time on it for now.

Reply:
I think it should be right. The configuration for costs is used to calculate both the cost benefit table, and the local costs. So one config generates two outcomes